pkg/activator/handler/metric_handler_test.go

```diff
@@ -107,11 +110,21 @@ func TestRequestMetricHandler(t *testing.T) {
 				}
 			}
 
-			handler.ServeHTTP(resp, req)
+			defer func() {
+				err := recover()
+				if test.wantErr && err == nil {
```

Comment:
I think wantPanic is a more suitable name here

Reply:
Done.